app/views/service/_svcs_show.html.haml

```diff
@@ -11,6 +11,10 @@
         - if retirement_job
           = miq_tab_header("retirement") do
             = _("Retirement")
+      -if @record.type == "ServiceAnsibleTower"
```

Comment:
@tumido wondering if we need `%w(ServiceAnsiblePlaybook ServiceAnsibleTower).include?(@record.type) ` check here and on line 41 below

Reply:
@h-kataria why would we? The tab headers for the `ServiceAnsiblePlaybook` are defined couple lines above. This "Job" tab is `ServiceAnsibleTower` specific.